### ISSUE TITLE
docs: Do you have a road map for the second half of the year?

### ISSUE THREAD

**Issue by cloud0072** [apitable/apitable#942]
Do you have a road map for the second half of the year?

Future roadmap planning is not updated on https://apitable.com/roadmap/ 
Will we continue to improve and update apitable?

**Comment by Alex-hufeng** [comment]
Of course, we will continue to update APITable, and we will release the plans for the second half of the year after sorting them out before the end of the month.

**Comment by mr-kelly** [comment]
This issue regarding the 2023 roadmap is now outdated. We continuously update our repository and features, and our long-term planning is reflected in our major releases. We will close this issue, but please continue to follow our community channels and GitHub releases for the latest updates.

---

这个关于 2023 年路线图的 Issue 现在已经过时了。我们不断更新我们的代码库和功能，我们的长期规划反映在我们的主要版本中。我们将关闭此 Issue，但请继续关注我们的社区渠道和 GitHub Releases 以获取最新更新。
